build.psm1

```diff
@@ -1135,88 +1105,89 @@ function Start-PSBootstrap {
     $sudo = if (!$NoSudo) { "sudo" }
 
     try {
-        # Update googletest submodule for linux native cmake
         if ($Environment.IsLinux -or $Environment.IsOSX) {
             try {
+                # Update googletest submodule for linux native cmake
```

Comment:
we should add a switch to bootstrap the components needed for native components so that we don't add them in systems where they are not needed.